aips/aip-reduce-stake-lockup-duration.md

```diff
@@ -0,0 +1,59 @@
+type: Core
+created: 7/11/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <mm/dd/yyyy>
+---
+
+# AIP-90 - Add Apple as a supported OIDC provider
+
+## Summary
+
+Aptos (APT), the layer 1 blockchain that we’re building upon, employs a Proof of Stake (PoS) consensus mechanism, wherein validators stake tokens to participate in network operations and governance.
+Currently, Aptos has a 30-day lock-up period for staked tokens, one of the longest among layer 1 protocols. This duration aims to ensure network stability and robust on-chain governance. However, feedback from various stakeholders suggests that reducing this period might offer significant benefits.
+As such, we’re proposing some possible implementation options for our community to consider. We conducted a survey within the community, and the winning option is to reduce the lock-up period to 14 days and the voting period to 3 days.
+
+## Rationale
+
+Several factors drive the consideration to reduce the lock-up period for staked APT tokens:
+
+- DeFi Partners and Projects Demands: Some of our DeFi partners, especially LSD and lending projects have expressed a desire for more flexible liquidity to facilitate more dynamic and responsive financial strategies.
+- Industrial Standard: CEX like OKX that offer staking products for multiple chains also favor shorter lock-up periods to better align with industry standards and improve their service offerings.(Other chain: Solana & Sui)
+- Community Demand: Our community members have also expressed their interest in actively participating in staking without committing to long lock-up periods. As such, reducing the lock-up duration could attract more users to stake their tokens, increasing overall network participation and security.
+- Efficient Operation: Node operators sometimes miss the unlock date, leading to inefficiencies. Shorter lock-up periods could mitigate these issues by providing more frequent opportunities to adjust stakes, enhancing operational flexibility.
+- Lowering Depeg Risk: Typically, when unstaking on Aptos, users wait for 30 days for their token return. To avoid this, some opt to swap to APT on other platforms, risking depeg and losing a significant amount of their original asset. Reducing the unstaking time can help shorten the waiting period, allowing users to unstake faster without worrying about depeg risks, thus maintaining a more stable peg for LSD tokens.
+
+## Impact and risks
+
+Despite the potential benefits, reducing the lock-up period introduces several risks:
+
+- Operational & Governance Challenges: Tracking lock-up dates becomes challenging with frequent release cycles and varying schedules among validators, limiting participation in governance. This can overwhelm validators and participants, potentially leading to burnout and reduced engagement. Such challenges may weaken governance effectiveness by shrinking the pool of active and informed participants.
```

Comment:
Potential impacted party includes:
- Aptos Foundation board: for reviewing the proposal and make a decision within 3 days. There's lot of prep work can be done beforehand, but it would be a change of how we review the source of truth and provide recommendations.
- Foundation delegated voters: have to react within 3 days to validate the proposal and cast the vote, which could lands in weekends too.
- Aptos dev team: need to coordinate with the development cycle, testing process, provide writeup to Foundation and inform the community to vote on proposals.
- All token holders who participates in governance voting: need to review and react to each proposal with shorter time, also shorter time window to voice concerns and reject proposals.